phpcs.xml

```diff
@@ -7,5 +7,8 @@
     </rule>
     <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingReturnTypeHint">
         <exclude-pattern>src/Logger/LineFormatter.php</exclude-pattern>
+        <exclude-pattern>src/Writer/Common.php</exclude-pattern>
+        <exclude-pattern>src/WriterInterface.php</exclude-pattern>
+        <exclude-pattern>src/Writer.php</exclude-pattern>
```

Comment:
that's practically the entire source of the library :-).  I think we can fix this by using return types in the block comments

Reply:
yeah you're right :) done

Reply:
ok, looks better, just remove these `excludes` and we're good